docs/pages/includes/database-access/pg-cancel-request-limitation.mdx

```diff
@@ -0,0 +1,40 @@
+### Unable to cancel a query
+
+If you use a PostgreSQL cli client like `psql`, and you try to cancel a query
+with ctrl+c, but it doesn't cancel the query, then you need to connect using a
+tsh local proxy instead.
+When `psql` cancels a query, it establishes a new connection without TLS
+certificates, however Teleport requires TLS certificates not only for
+authentication, but also to route database connections.
+
+Start a tsh local proxy using `tsh proxy db` and then connect to the local proxy
+with `psql`.
```

Comment:
I feel we should mention this also works with `tsh db connect` when TLS routing is enabled. People are not loving `tsh proxy db` unless they _really_ have to use it (like a GUI).

We can optionally mention Teleport Connect too which always uses a local proxy.

Reply:
I explain how `tsh db connect` works if TLS routing is enabled and how Teleport Connect is an alternative as well. Also linked info on each.

 [672fcfd](https://github.com/gravitational/teleport/pull/38352/commits/672fcfd6be5d24cad838eb5971ae1e5a0b5ce98d) 
(pushed the changes in the same commit as my redshift changes)